### DISCUSSION TITLE
[Feature] Copy open-pdf link

### DISCUSSION THREAD

**Discussion by mjthoraval** [windingwind/zotero-actions-tags#191] [Ideas]
I would like to copy the open-pdf link from a righ-click on an item.
It should open the same file as a double-click on the item would do.
Can anyone help on this?
Thank you very much!

It was mentionned here: https://forums.zotero.org/discussion/comment/447114#Comment_447114
But the link provided in there only gives a `select` link.
See for example: https://forums.zotero.org/discussion/86980/quickly-create-a-open-pdf-link

**Comment by windingwind** [comment]
https://github.com/windingwind/zotero-actions-tags/discussions/115
